reporter/src/main/kotlin/utils/AttributionDocumentPdfModel.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.reporter.utils
+
+import org.eclipse.sw360.antenna.attribution.document.core.model.ArtifactAndLicense
+import org.eclipse.sw360.antenna.attribution.document.core.model.LicenseInfo
+
+import java.util.Optional
+
+data class AttributionDocumentPdfModel(
```

Comment:
Rename to `AntennaAttributionDocumentModel`, similar to above. The `Pdf` in the name seems unnecessary to me, or does Antenna also support other formats you plan to add later?

Reply:
Antenna only supports PDF output, and we have no plans to change that. But as mentioned above the goal was to emphasize the PDF part.

Reply:
I am not sure, there is also the need to provide the document as html, which is also available within Antenna.

Reply:
Oh, it's new to me that Antenna can also produce HTML attribution documents. Is there yet more formats besides PDF and HTML? What would it take to also support HTML here, should this be implemented as another option to the reporter, or as a whole new reporter?